examples/rbd/template-ceph-cluster-ID-secret.yaml

```diff
@@ -0,0 +1,36 @@
+---
+# This is a template secret that helps define a Ceph cluster configuration
+# as required by the CSI driver. This is used when a StorageClass has the
+# "clusterID" defined as one of the parameters, to provide the CSI instance
+# Ceph cluster configuration information.
+apiVersion: v1
+kind: Secret
+metadata:
+  # The <cluster-id> is used by the CSI plugin to uniquely identify and use a
+  # Ceph cluster, the value MUST match the value provided as `clusterID` in the
+  # StorageClass
+  name: ceph-cluster-<cluster-id>
+  namespace: default
+data:
+  # Base64 encoded and comma separated Ceph cluster monitor list
+  #   - Typically output of: `echo -n "mon1:port,mon2:port,..." | base64`
+  monitors: <BASE64-ENCODED-MONLIST>
+  # Base64 encoded and comma separated list of pool names from which volumes
+  # can be provisioned
+  pools: <BASE64-ENCODED-POOLIST>
```

Comment:
can we create volume in multiple pools?  what's the advantage of having pool list here?